website/content/docs/ui/custom-messages/index.mdx

```diff
@@ -0,0 +1,88 @@
+---
+layout: docs
+page_title: Manage custom messages
+description: >-
+  Use custom messages in the Vault UI to share system-wide alerts.
+---
+
+# Manage custom messages in the Vault UI
+
+Use custom banners and modals in the Vault UI to share system-wide alerts for all Vault UI users. Users can then dismiss messages by clicking "X" on a banner or "Confirm" for a modal.
+![Vault UI sidebar. The "Custom Messages" button in Settings section is outlined with a red box](/img/custom-messages-sidebar.png)
```

Comment:
@schavis we discussed adding a sentence or two that messages created aren't permanently.

Use custom banners and modals in the Vault UI to share system-wide alerts for all Vault UI users. Users can then dismiss messages by clicking "X" on a banner or "Confirm" for a modal. Clicking dismiss will not permanently dismiss the banner or modal. Users will see the message if they refresh the page or log out of their current session.
![Vault UI sidebar. The "Custom Messages" button in Settings section is outlined with a red box](/img/custom-messages-sidebar.png)